### DISCUSSION TITLE
My firgate and HA integration seems to be broken, clips don't show

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#15188] [Ask A Question]
So as usual, HA works great for me except when I want to show it off to people.  Yesterday I was trying to show off how great HA is with the Frigate integration, only to find out it seems to be broken.

About a month or two ago I realized that on the frigate card if I click on the side of the video it showed a list of clips/recordings for that camera.  It still does that, but when I try to see the video nothing shows.  I then went to Media/Frigate/Clips folder, and I can see the list of clips, but when I try to play them, they don't play.

Any ideas on how I can fix this?

![image](https://github.com/user-attachments/assets/3e68f4ba-2ea1-459e-a92f-df6b433cedbc)

![image](https://github.com/user-attachments/assets/2522a01f-390f-46b0-81db-940f21bbd48a)

As you can see it does list the clips, and states how long the clip is, but it won't play it

**Comment by NickM-27** [comment]
This is an HA bug

**Reply by CancunManny** [reply]
Do you know if it is related to one of the recent updates?

I did a restore to Nov 17th, and seems but was there already.  Seems I have to keep more backups.

**Reply by NickM-27** [reply]
2024.11

**Reply by CancunManny** [reply]
Am I correct to assume you guys already reported the bug to the HA team so no real need for me to formally try to report the bug myself?
